doxygen-awesome.css

```diff
@@ -186,7 +187,7 @@ html {
         color-scheme: dark;
 
         --primary-color: #1982d2;
-        --primary-dark-color: #86a9c4;
+        --primary-dark-color: #ddb582;
```

Comment:
The new colors that you choose for WxWidgets look great IMO! I'd like to keep the original colors as default though, for both variables, `--primary-dark-color` and `--fragment-desc-background`.